package.json

```diff
@@ -11,6 +11,10 @@
   "files": [
     "lib"
   ],
+  "scripts": {
+    "pretest": "eslint lib --fix",
+    "test": "./node_modules/.bin/grunt test"
```

Comment:
We don't use build tools like grunt or gulp for components anymore as you can specify the npm test command to execute your test framework directly

````
scripts: {
    "test": "NODE_ENV=test ./node_modules/mocha/bin/mocha --recursive spec"
}
````